staging/src/github.com/kubeedge/mapper-framework/_template/mapper/cmd/main.go

```diff
@@ -9,27 +9,24 @@ import (
 	"github.com/kubeedge/Template/device"
 	"github.com/kubeedge/Template/pkg/common"
 	"github.com/kubeedge/Template/pkg/config"
-	"github.com/kubeedge/Template/pkg/grpcclient"
 	"github.com/kubeedge/Template/pkg/grpcserver"
 	"github.com/kubeedge/Template/pkg/httpserver"
 	"github.com/kubeedge/Template/pkg/util/parse"
 )
 
 func main() {
 	var err error
-	var c config.Config
+	var c *config.Config
```

Comment:
Now c is a pointer. In order to pass parameters correctly, panel.DevInit should pass in c directly instead of &c.
![123](https://github.com/kubeedge/kubeedge/assets/48212997/745094a7-49b2-4a77-9c65-05c1db3c8655)

Reply:
ok, I'll fix it.